src/Commands/Admin.py

```diff
@@ -1,10 +1,12 @@
 from CheckForUpdates import updateBot
 from EmbedHelper import AdminEmbed, ErrorEmbed, QueueUpdateEmbed
 from typing import List
-from bot import __version__
-from discord import Role, Embed, Member
+from Types import Team
+from bot import __version__, LB_CHANNEL
+from discord import Role, Embed, Member, channel
```

Comment:
Can you alias channel as Channel like we do in SixMans? The capitalization makes it seem more like a type and is consistent with the other discord types.